flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -31,7 +31,13 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
 
     changePage(_pageName) {
         let pageName = _pageName.toUpperCase();
+        const isOnGround = SimVar.GetSimVarValue("GEAR IS ON GROUND","Bool");
         for (let i = 0; i < this.lowerScreenPages.length; i++) {
+            //skip CRZ and go back to ENG Page
+            if (this.lowerScreenPages[i].name == "CRZ" && isOnGround) {
+                pageName = "ENG";
+                i = 0;
```

Comment:
Wouldn't it be better to avoid a second run through the list?
We can store the ENG-index and if it is not set, we continue with the current loop.

It looks like the second run can be easily avoided